spec/features/message_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+describe 'Messages', :type => :feature do 
+  describe 'New Message' do
+      let!(:user) {create :user}
```

Comment:
The github Ruby styleguide states that spaces after the opening and before the closing curly brace, are preferred